imports/plugins/included/surcharges/server/no-meteor/startup.js

```diff
@@ -0,0 +1,47 @@
+import xformCartGroupToCommonOrder from "/imports/plugins/core/cart/server/no-meteor/util/xformCartGroupToCommonOrder";
+import getSurcharges from "./getSurcharges";
+
+const EMITTED_BY_NAME = "SURCHARGES_PLUGIN";
+
+/**
+ * @summary Called on startup
+ * @param {Object} context Startup context
+ * @returns {undefined}
+ */
+export default function startup(context) {
+  const { appEvents, collections } = context;
+  const { Cart } = collections;
+
+  // Adds Surcharges collection to context
+  collections.Surcharges = context.app.db.collection("Surcharges");
```

Comment:
Did you look at the queries being made to see if you should call `collectionIndex` here to add some?

Reply:
It seems like `_id` might be the only index needed... I'm assuming that's probably already indexed though?